app/presenters/publishing_api_presenters/coming_soon.rb

```diff
@@ -29,7 +29,9 @@ def as_json
       locale: locale,
       update_type: 'major',
       details: { publish_time: edition.scheduled_publication.as_json },
-      routes: [ { path: base_path, type: "exact" } ]
+      routes: [ { path: base_path, type: "exact" } ],
+      # We don't store when the coming_soon was first created, so just use the current time
+      public_updated_at: Time.zone.now,
```

Comment:
We should be able to use the last updated time of the scheduled edition, since that should map to when the item was scheduled.